examples/with-why-did-you-render/README.md

```diff
@@ -29,7 +29,33 @@ You can see `why-did-you-render` console logs about this redundant re-render in
    }
    ```
 
-1. import `scripts/wdyr.js` as the first import of `_app`.
+1. Import `scripts/wdyr.js` as the first import of `_app`.
+
+1. Make sure that [`react-preset`](https://babeljs.io/docs/en/babel-preset-react) uses `@welldone-software/why-did-you-render` to import the monkey patched `React` with WDYR, by modifying `next/babel` in `babel.config.js`:
```

Comment:
```suggestion
2. Make sure that [`react-preset`](https://babeljs.io/docs/en/babel-preset-react) uses `@welldone-software/why-did-you-render` to import the monkey patched `React` with WDYR, by modifying `next/babel` in `babel.config.js`:
```

Reply:
It's actually not needed. Markdown knows to resolve correct counting by itself.
I think it's even better to only use `1.` since you can always add points in the middle without breaking the count.

Reply:
![image](https://user-images.githubusercontent.com/5188459/108476250-7d70b300-729a-11eb-8c58-b443d4369453.png)